_episodes/08-hygiene.md

```diff
@@ -84,6 +84,20 @@ the tool shapes the hand that shapes the tool.
 > *   87 Turing Avenue
 > *   January 25, 1971
 > *   the XY coordinate (0.5, 3.3)
+>
+> > ## Solution
+> > New Zealand is the only clear-cut atomic value.
+> >
+> > The address and the XY coordinate contain more than one piece of information
+> > which should be stored separately:
+> > - House number, street name
+> > - X coordinate, Y coordinate
+> >
+> > The date entry is less clear cut.
+> > There is a `DATE` datatype in SQL, so dates could be considered atomic.
+> > However, there might be situations where we want to keep the month, day, and year
+> > as separate variables.
```

Comment:
I'm sorry, but I think some of those situations should be listed out if you can think of any. Otherwise I don't think we should advocate for representing values in anything other than their native SQL type.

I worry that because we don't give any detail here, people will decide that their own situation is one of those, and create weird schemas that will cause them (and their colleagues) pain down the line.

Reply:
I agree that the `DATE` type should be used for dates. However, the question asks whether the examples are atomic, not which data type should be used to store them in SQL. Dates are arguable either way I would say -- see the article linked to by @henrykironde.

I think the question either needs rewording, or there should be something of a discussion to say date is neither clear-cut atomic nor non-atomic, but there is a date type that should be used for it.

Does that sound ok?

Reply:
I agree that it's worth explicitly stating what is meant by "atomic" here, or even replace with a better word. ["Atomic" in a database context](https://en.wikipedia.org/wiki/Atomicity_(database_systems)) usually refers to transactions (the A in ACID).

However I still maintain that it is commonly accepted that dates are atomic in a database normalization context ([see for example this answer](https://stackoverflow.com/questions/24029620/what-is-atomicity-in-dbms)).

Reply:
Breaking down names to first-middle-last is also appropriate in some context (e.g. government forms) but not generally (first-middle-last is a very US-centric view of names and is very inappropriate for a lot of countries).